supervisor/hyperpod.go

```diff
@@ -366,7 +366,12 @@ func (hp *HyperPod) startNsListener() (err error) {
 
 func (hp *HyperPod) stopNsListener() {
 	if hp.nslistener != nil {
-		hp.nslistener.cmd.Process.Kill()
+		go func() {
+			hp.nslistener.cmd.Process.Kill()
+		}()
+		if err := hp.nslistener.cmd.Process.Signal(syscall.Signal(0)); err == nil {
```

Comment:
It is strange that Process.Signal() and Process.Kill() are called at the same time.
if Process.Kill() is needed when Process.Signal() is ignored by the target, I think a timer is also needed before Process.Kill().